server/db-patches/patch-2-3.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE data ADD url TEXT;
```

Comment:
Ok, I understand. Create the column first, do the update, and then alter the column to set not null. Cool.